pipelines.py

```diff
@@ -743,6 +745,33 @@ def rnn_ensemble(config, is_train):
     return output
 
 
+def get_text_features(train_meta, test_meta):
```

Comment:
@jakubczakon in the future and in Steps, consider refactoring, since it is not `Step` in pipeline.